devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -69,7 +69,7 @@ metadata {
 		input "temperatureReportInterval", "enum", title: "Temperature Report Interval",
 				options: ["reports inactive", "5 minutes", "15 minutes", "30 minutes", "1 hour", "6 hours", "12 hours", "18 hours", "24 hours"], defaultValue: "${temperatureReportInterval}", displayDuringSetup: true
 		input "temperatureReportHysteresis", "number", title: "Temperature Report Hysteresis", description: "Available settings: 1-100 C", range: "1..100", displayDuringSetup: true
-		input "temperatureThreshold", "number", title: "Overheat Temperature Threshold", description: "Available settings: 0 or 2-100 C", range: "0..100", displayDuringSetup: true
```

Comment:
Is 1 valid for this preference?

Reply:
All of the code for the preference leads me to believe that it is.

Reply:
Saying the range is 0 -100 seems more natural. But then we'll need to get updated translations so maybe we should just remove the description since we already display the valid range.

Reply:
I think the intention of the text is to make it clear that a setting of 0 behaves in a way that is not linearly consistent with 1-100.

Reply:
What does 0 do? It's not clear from the description how it's different

Reply:
I believe 0 means that the device will never report overheat.

Reply:
<img width="255" alt="Screen Shot 2020-07-18 at 2 52 32 PM" src="https://user-images.githubusercontent.com/144884/87860847-5fa72e00-c906-11ea-89ab-bc22e9d959d1.png">

Only 1-100 is valid. The current code treats a value less than 1 as 1.